netlify.toml

```
@@ -1,33 +1,36 @@
 [Settings]
-# Added automatically by the Netlify CLI. It has no effect during normal 
+# Added automatically by the Netlify CLI. It has no effect during normal
 # Git-backed deploys.
 ID = "48a8adb4-6837-44ba-a9c4-267b56de8212"
 
-# Settings in the [build] context are global and are applied to all contexts 
-# unless otherwise overridden by more specific contexts.  
+# Settings in the [build] context are global and are applied to all contexts
+# unless otherwise overridden by more specific contexts.
 [build]
-  # Directory to change to before starting a build. 
+  # Directory to change to before starting a build.
   # This is where we will look for package.json/.nvmrc/etc.
-  base = "docs/"
+  base = "docs"
 
-  # Directory (relative to root of your repo) that contains the deploy-ready 
+  # Directory (relative to root of your repo) that contains the deploy-ready
   # HTML files and assets generated by the build.
-  publish = "docs/_site"
+  publish = "docs/build"
```

This should be aligned with https://github.com/arcus-azure/arcus.backgroundjobs/blob/master/netlify.toml#L15
```suggestion
  publish = "build"
```